### ISSUE TITLE
HTTPServiceUnavailable on endpoint not available

### ISSUE THREAD

**Issue by jamielennox** [cloudendpoints/endpoints-management-python#73]
When endpoint-management is reaching out to get service config i'm getting a 403, I'm still unsure what i've changed to get this but it doesn't really matter for now. [This raises a ServiceConfigException](https://github.com/cloudendpoints/endpoints-management-python/blob/f9d4cda808bbd79bd4d7bcb63d624b102e12ec98/endpoints_management/config/service_config.py#L81-L85).

This gets caught at the wsgi init layer, but instead of propogating the error it [returns a webob.exc.HTTPServiceUnavailable](https://github.com/cloudendpoints/endpoints-management-python/blob/f9d4cda808bbd79bd4d7bcb63d624b102e12ec98/endpoints_management/control/wsgi.py#L123-L130).

What this means is that in my entrypoint file like: 

```
api = endpoints.api_server([MyAPI])
```

the `api` object is now a `HTTPServiceUnavailable` object and endpoints tries to route traffic to it ending up with errors that look like: 
```
Traceback (most recent call last):
  File "/base/alloc/tmpfs/dynamic_runtimes/python27g/7894e0c59273b2b7/python27/python27_lib/versions/1/google/appengine/runtime/wsgi.py", line 267, in Handle
    result = handler(dict(self._environ), self._StartResponse)
  File "/base/data/home/apps/f~teg-jamie-testing/listings:20180625t231203.410697777093027702/vendor/webob/exc.py", line 271, in __init__
    self.headers.extend(headers)
  File "/base/data/home/apps/f~teg-jamie-testing/listings:20180625t231203.410697777093027702/vendor/webob/multidict.py", line 233, in extend
    for k, v in other:
TypeError: 'instancemethod' object is not iterable
```
because the 2nd argument to `HTTPServiceUnavailable.__init__` is headers, not `start_response` as expected.

I'm not entirely sure what good behaviour here is, it makes sense that this needs to return an error but i really want it to happen before i start serving requests (actually i want to fail the deployment based on it).

My current solution i think is going to be like (pseudo): 
```
_api = endpoints.api_server([MyAPI])
if isinstance(api, webob.exc.Exception): 
    def fail(environ, start_response):
        logging.warning("Failed to start API", exc_info=_api.something())
        # return 503 with some actual information
        
    api = fail
else:
    api = _api 
```
so that at least I get a traceback that I can debug failures with because the current error tells me nothing.

Thanks

**Comment by inklesspen** [comment]
The problem is that I needed to instantiate the HTTPServiceUnavailable class instead of returning the class itself. Thanks for catching this!

**Comment by inklesspen** [comment]
Fixed in 1.9.1.
